.github/workflows/build-test.yml

```diff
@@ -162,18 +152,7 @@ jobs:
 
           echo 'Checking nproc version.'
           nproc --version
-      - name: Set up Conan home directory (MacOS)
-        if: ${{ inputs.os == 'macos' }}
-        run: |
-          echo 'Setting up Conan home directory.'
-          export CONAN_HOME=${{ github.workspace }}/.conan
```

Comment:
This is actually a bug on macOS as well, since this env variable won't exist in the next steps:
https://github.com/XRPLF/rippled/actions/runs/17263009674/job/49000796369#step:13:10

`Copying file default to /Users/ec2-user/.conan2/profiles/.`

And since macOS runners are reused, this creates all sorts of problems